src/util/ChartUtils.ts

```diff
@@ -421,7 +423,7 @@ const isErrorBarRelevantForAxis = (layout?: LayoutType, axisType?: AxisType, dir
 
 export const getDomainOfErrorBars = (
   data: any[],
-  item: any,
+  item: { props: { children: any } },
```

Comment:
Good question, this is automatically inferred from usage, and only marginally better than any.

Reply:
I could also just revert this change.

Reply:
I can't stand `any`s lol. I like the direction but it looks like we have two usages of this from other helper functions and then usages of those somewhere else. If we can prove this doesn't get called with other props then I'm cool with it else lets address in a different issue

Reply:
Looks like one of those usages ends up here https://github.com/recharts/recharts/blob/master/src/chart/generateCategoricalChart.tsx#L326

That doesn't look safe to me being that `graphicalItems` is `any` and probably has more props

Reply:
though now that I think about it having the type there shouldn't break anything at run time (since JS) so as long as it doesn't break anything at `tsc`/build time it should be fine

Reply:
I reverted the change, lets improve the type properly in a separate PR. This change does not add much value.